src/XMakeBuildEngine/Evaluation/ProjectParser.cs

```diff
@@ -187,61 +187,6 @@ private void ParseProjectElement(XmlElementWithLocation element)
             // so we have to set it now
             _project.SetProjectRootElementFromParser(element, _project);
 
-            if (element.HasAttribute(XMakeAttributes.sdk))
```

Comment:
Shouldn't this method (and the other thing that can have SDKs, ParseImportElement too) do some SDK string validation, such as ensure it's not empty?